cairo/src/helper.rs

```diff
@@ -0,0 +1,99 @@
+//! This module inlcudes some field helpers that are useful for Cairo
+
+use ark_ff::FftField;
+use o1_utils::FieldHelpers;
+
+//(TODO move to utils inside FieldHelpers)
+
+/// Field element helpers for Cairo
+pub trait CairoFieldHelpers<F: FftField> {
```

Comment:
can you remove the trait bound here on `F`? Not sure if that makes sense

Reply:
I can try! I thought they had to be specified both in the trait and the impl, but I guess it's more flexible if you only specify the generic at the impl level... Cool feature.